### ISSUE TITLE
ST Anything Device no longer responding

### ISSUE THREAD

**Issue by gshanson** [DanielOgorchock/ST_Anything#206]
I hope someone can  help with this issue.
I have had a ST Anything device working for for a few months.
I am using ST_Anything_Multiples_ESP8266WiFi.
Then a few weeks ago, the device stop updating the values.
The ST app says the device has not recent activities.
I tried rebooting the ESP8266 several times and there has been no change.
Today I create a new device in the Smartthings IDE and configured it in the ST App.
I restarted the ESP8266 and ... nothing.
It does not create the child devices.

Any ideas on how I can restore the operation of my ESP8266 ST Anything device?
It was working fine until a few weeks ago.

I would like to get the original device working or create a new working device.

Thank you in advance for your help.

**Comment by ogiewon** [comment]
Has the IP address of your ST Hub changed?  Have you tried rebooting your ST Hub?  Can you ping both the ST Hub and the ESP8266 via their IP addresses from another computer?

**Comment by gshanson** [comment]
Hello Daniel,

Thank you for your prompt response.
In fact the IP address of my ST Hub had changed.  I corrected that issue and my ESP8266 device is now working again.

As a side note, the new ESP device that I created still fails in creating the child devices.  It give me errors on the ST App.  This is not a problem for me at this time since my original device is working.  The original parent and its’ child devices were created some months ago.
Sometime in the future I may need to pursue this problem, but for now, all is well.

Thanks Again,
Glenn
